### DISCUSSION TITLE
Hello just wanting help. New to all this.

### DISCUSSION THREAD

**Discussion by Manifestingmulla** [community/community#45757] [New to GitHub]
What do I need to be doing. I'm working through a smartphone

**Comment by jge162** [comment]
what are you needing help with on GitHub.

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @Manifestingmulla  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
